composer.json

```diff
@@ -1,20 +1,13 @@
 {
-    "name": "laravel-zero/laravel-zero",
-    "description": "The Laravel Zero Framework.",
-    "keywords": ["framework", "laravel", "laravel zero", "console", "cli"],
-    "homepage": "https://laravel-zero.com",
+    "name": "ossindex/bach",
+    "description": "Dependency vulnerability auditor for PHP",
+    "homepage": "https://github.com/OSSIndex/bach",
     "type": "project",
-    "license": "MIT",
+    "license": "Apache-2.0",
```

Comment:
Do we also want to include the license file itself, or is it fine to just ref it here?

Reply:
It's already there :)
https://github.com/OSSIndex/bach/blob/master/LICENSE

Reply:
> It's already there :)
> https://github.com/OSSIndex/bach/blob/master/LICENSE

I see :) Ok cool, looks good to me!